install_files/securedrop-config/DEBIAN/postinst

```diff
@@ -15,6 +15,7 @@ case "$1" in
     if [ -f "$apt_security_list" ]; then
         sed -i 's/deb\.torproject\.org\/torproject\.org/tor-apt.freedom.press/g' "$apt_security_list"
     fi
+    sudo apt remove linux-image-generic-lts-xenial linux-image-.*generic
```

Comment:
Nit: Needs `-y`, otherwise will hang forever. We also probably need to single-quote the args to prevent the shell from expanding the regex special chars in globs.